Feature

tta2yta opened the pull request:
We created a branch called "feature" for the development of The Next Web clone. 
On the initial commit, we created a basic structure for the page and added CDN links Font awesomely, set up linter, configured it and I also added Google Fonts.
The "Mobile first" approach was used first to achieve responsiveness principles.
Elements were positioned using the grid, flex, and positioning techniques.
The issues raised from the first review of the project in the dashboard are fully addressed.

This project was approved last week but due to some challenges that occurred with the professional skills task, The student success team informed us to repeat it this week.  Could you please review it soon so that we can proceed to the week 3 project as we have already started it by money and we did half part of it?

Please review our project before we merge my branch.
Thank you in advance!

prolajumokeoni committed:
initial commit

tta2yta committed:
initial commit

tta2yta committed:
adding refernce link to head

tta2yta committed:
adding nav1 section

tta2yta committed:
adding nav2 section

tta2yta committed:
adding nav3 section

tta2yta committed:
adding styles to nav1 section

tta2yta committed:
adding styles to nav and nav3 section

tta2yta committed:
adding styles to nav and nav3 section

tta2yta committed:
adding styles and media quries to nav and nav3 section

prolajumokeoni committed:
adding header image

prolajumokeoni committed:
add funding section

prolajumokeoni committed:
latest new header

prolajumokeoni committed:
add latest news images

prolajumokeoni committed:
add more latest funding information

tta2yta committed:
adding latest funding images  section

tta2yta committed:
adding popular news section

tta2yta committed:
adding latest deals section header

tta2yta committed:
adding latest deals section body

tta2yta committed:
adding footer section

tta2yta committed:
adding styles to serac-icon and nav section

tta2yta committed:
adding styles to bo1 box 2 box 3main-grid1

tta2yta committed:
adding gradient to box1

tta2yta committed:
adding media qiueris for tablet

tta2yta committed:
adding styles to latest news section

tta2yta committed:
adding media quries for main2-grid and para-box

tta2yta committed:
adding styles for funding section

tta2yta committed:
adding styles for funding section

tta2yta committed:
adding styles for popular news section

tta2yta committed:
adding styles for footer section

tta2yta committed:
Update README.md

tta2yta committed:
Update README.md

tta2yta committed:
Update README.md

tta2yta committed:
adding readme

tta2yta committed:
Merge branch 'feature' of https://github.com/prolajumokeoni/The-Next-Web into feature

prolajumokeoni committed:
Update README.md

edit readme

tta2yta committed:
Update README.md

tta2yta committed:
correcting linters

tta2yta committed:
Merge branch 'feature' of https://github.com/prolajumokeoni/The-Next-Web into feature

tta2yta committed:
correcting linters

tta2yta commented:
we have fixed the linters properly in our local computer but in GitHub, it is rasing an exception of Error: Cannot find module 'postcss/lib/input'.

MuriloRoque reviewed (changes requested):
## Status: Changes requested :red_circle: 

### Hello @tta2yta and @prolajumokeoni :wave: 

### You have done an amazing job on this project, it's really good, congratulations :clap: 

### There are only some issues we need to address before moving on, please check my suggestions below:

- [ ] The images on your main 3 articles are not showing, the console informs that they are not being found on your project. Are you sure that the path to them is right on your HTML? Or maybe it's another issue? 

**Screenshot:**

![Screenshot from 2021-01-27 13-50-35](https://user-images.githubusercontent.com/59691171/106026200-48f94500-60a8-11eb-8ade-d090173b0483.png)

- [ ] Add some margin between your sections, they are looking too close to each other, especially on big screens. Check the image below:

![Screenshot from 2021-01-27 13-51-07](https://user-images.githubusercontent.com/59691171/106026276-63cbb980-60a8-11eb-8c76-fe55860464b9.png)

- [ ] Your linters check is failing, please fix the issues.

**To Highlight :bulb:**

- Your project is well structured;
- Your page is very detailed, almost close to the original;
- Your code is clean, which is a very nice quality.

Kindly make the requested changes and submit them for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[![Twitter Follow](https://img.shields.io/twitter/follow/MuriloRoquePai1?label=Follow%20Murilo%20on%20Twitter&style=social)](https://twitter.com/MuriloRoquePai1)

tta2yta committed:
adding top margin to funding section

tta2yta committed:
Update README.md

tta2yta commented:
Hello.  Thank you for your comments. We have updated the points you mentioned. Could you please check it again. Thank you for your time.

umairarshadbutt commented on css/style.css:
```diff
@@ -0,0 +1,1254 @@
+    max-width: 100%;
+    text-align: center;
+
+    /* padding-right: 400px; */
```
Remove the commented CSS

umairarshadbutt commented on index.html:
```diff
@@ -0,0 +1,732 @@
+                        <li><a href="">Spaces</a> </li>
+                        <li><a href="">Deals</a> </li>
+                        <li><a href="">Answers</a> </li>
+                        <li><a href="">TNW Program</a> </li> 
```
Add `#` at empty `href` attributes

umairarshadbutt commented on index.html:
```diff
@@ -0,0 +1,732 @@
+                <div class="funding-box">
+                   <a href="#"><img class="funding-image" src="./img/3.1.png" alt=""></a> 
+                    <p class="top1"><a href="#">BharatPe</a></p>
+                    <h3 class="price"><a href="">₹2B</a></h3>
```
Add `#` at empty `href` attributes

umairarshadbutt commented on index.html:
```diff
@@ -0,0 +1,732 @@
+            <div class="main1-grid-cont">
+                 <div class="box1 grid-class">
+                     <a class="" href="https://thenextweb.com/neural/2021/01/19/us-regulator-approves-first-ever-fully-autonomous-commercial-drone-flights/">
+                        <img class="logo-inside-grid" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="">
```
Add the description in the `alt` attribute.

umairarshadbutt commented on index.html:
```diff
@@ -0,0 +1,732 @@
+        <div class="main-flex flex">
+            <div class="main2-grid-cont">
+                 <div class="grid-boxes">
+                     <a href="#"><img src="./img/1.1.jpg" alt="" class="image-box"></a>
```
Add the description in the `alt` attribute.

umairarshadbutt reviewed (changes requested):
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @prolajumokeoni 
Such a remarkable project you made :+1: ! Awesome README :100: !
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [ ] Fix the linter issues.
- [ ] Remove the commented CSS.
- [ ] Add `#` at empty `href` attributes. (`href` attribute is empty at 28 times in HTML document)
- [ ] Add the description in the `alt` attribute.(`alt` attribute is empty at 28 times in HTML document)
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

tta2yta committed:
adding text to alt attribute

tta2yta committed:
Merge branch 'feature' of https://github.com/prolajumokeoni/The-Next-Web into feature

tta2yta commented:
Hello. Thank you for your comments. We have updated the points you mentioned. Could you please check it again. Thank you for your time.

we have fixed the linters properly in our local computer but in GitHub, it is rasing an exception of Error: Cannot find module 'postcss/lib/input'.

descholar-ceo commented on README.md:
```diff
@@ -1,2 +1,55 @@
+- <a href="https://www.w3schools.com/" target="_blank">w3schools.com</a>
+- <a href="https://www.theodinproject.com/" target="_blank">The Odin Project</a>
+- <a href="https://www.stackoverflow.com/" target="_blank">Stackoverflow</a>
+- <a href="https://css-tricks.com/" target="_blank">CSS-TRICKS</a>
```
I would advise you to use markdown language for readme file, not HTML

prolajumokeoni replied:
We have implemented this, thank you!

tta2yta committed:
reinstalling lintel node modules

descholar-ceo commented:
# This Pull Request is under review by [descholar-ceo](https://github.com/descholar-ceo) :eyes: :eyes: 
## Please wait a bit ...

descholar-ceo commented on README.md:
```diff
@@ -1,2 +1,55 @@
+- Github: [Tedros Tesfay](https://github.com/tta2yta)
+- Emaii: tta2yta@gmail.com
+
+👤
```
Add a name of the second author

descholar-ceo commented on README.md:
```diff
@@ -1,2 +1,55 @@
+👤 **Oni Olajumoke Priscilla and Tedros Tesfay**
+
+- Github: [Tedros Tesfay](https://github.com/tta2yta)
+- Emaii: tta2yta@gmail.com
```
Is it `Emaii`? rather than being `Email` ?

prolajumokeoni replied:
We added links to our Linkedin accounts and removed the emails.

descholar-ceo commented on README.md:
```diff
@@ -1,2 +1,55 @@
-# The-Next-Web
-A clone application of the next web original website
+
+# HTML/CSS capstone project
```
Is this a capstone? Your capstone is yet to come!

descholar-ceo commented on README.md:
```diff
@@ -1,2 +1,55 @@
+
+# HTML/CSS capstone project
+
+This project is a responsive website application developed as a completion of HTML & CSS section of Microverse main curriculum. It is a collobtative project intended to showcase HTML & CSS Skills. It is a real-world-like project built with some business specifications. It is a clone of the thenextweb website. It consist of only one page, that is home page. It is adaptive to different standard scr ...
```
This text contains some grammatical and typing errors, please fix them by inspecting the text with [Grammarly](https://app.grammarly.com/), you will be able to spot those errors

prolajumokeoni replied:
We made some edits to the whole readme file, we hope to hear from you soon, Thank you

descholar-ceo replied:
Well done @prolajumokeoni , keep up the good work :muscle: :muscle: :muscle:

descholar-ceo reviewed (changes requested):
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @tta2yta and @prolajumokeoni  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [ ] Fix the typos errors from the readme file, you can use [Grammarly](https://app.grammarly.com/) to spot them
- [ ] TWN logo disappears from the top three articles (the leftmost big article) on medium screens 

| Your clone | Original page |
| --- | --- |
| ![Screenshot from 2021-01-28 10-10-55](https://user-images.githubusercontent.com/56886064/106108610-27a06380-6151-11eb-9f61-0b92f2cc611b.png)|![Screenshot from 2021-01-28 10-11-55](https://user-images.githubusercontent.com/56886064/106108694-4999e600-6151-11eb-81fd-c5501efb836d.png) |

:point_right: Could you see the difference :question: :question: :question: 
___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

tta2yta committed:
Update README.md

tta2yta committed:
Update README.md

tta2yta committed:
Update README.md

tta2yta committed:
Update README.md

tta2yta committed:
adding style to top header images

tta2yta committed:
Merge branch 'feature' of https://github.com/prolajumokeoni/The-Next-Web into feature

tta2yta commented:
Hello. Thank you for your comment. We have fixed all the points you mentioned regarding the readme file. Regarding the "TWN logo disappears " you said it was not disappearing before in the original website. But we did it as you said. Thank you for your time.

prolajumokeoni committed:
Update README.md

edit readme

prolajumokeoni committed:
Update README.md

edit readme

prolajumokeoni committed:
Update README.md

edit readme

prolajumokeoni committed:
Update README.md

edit readme

prolajumokeoni committed:
Update README.md

edit links in readme

prolajumokeoni committed:
Update README.md

edit readme links

prolajumokeoni committed:
Update README.md

edit readme content

prolajumokeoni committed:
Update README.md

edit readme links

prolajumokeoni committed:
Update README.md

edit readme

elmejdki reviewed (approved):
# status: approved :heavy_check_mark: 

Great job @team, you made a cool clone,

now you can merge your project is approved :checkered_flag: 

## Optional Changes:
but before merging can you please fix this optional requirement below:
:point_down: 

- [ ] Please fix your media queries cause in tablet resolutions it's showing up as if it's on a phone:
![image](https://user-images.githubusercontent.com/34653764/106133827-0e55e200-6166-11eb-8bfb-d2d5c6cfe7ef.png)

## Happy Coding :tophat: 
## Zakariae El Mejdki
